nat/traversal/pinger.go

```diff
@@ -177,7 +180,7 @@ func (p *Pinger) PingProvider(ip string, port int) error {
 	}
 
 	// wait for provider to setup NAT proxy connection
-	time.Sleep(200 * time.Millisecond)
+	time.Sleep(400 * time.Millisecond)
```

Comment:
We are tweaking 3rd time with no explanation. Maybe there could be a better check?

Reply:
Theres little bit of tweaking here since we are not sure if proxy has been setup for current session. If consumer starts to send protocol (ssh) messages without proxy set up, connect will fail. 
We might be able to implement another round trip of messaging between consumer - provider confirming that provider is ready to receive protocol connect, but at this point it looks to be excessive still. Later it should be implemented probably. Still I am hoping to find some sane value here, its not nice, but should suffice atm.